docs/content/guides/getting-started/saving-data.md

```diff
@@ -19,7 +19,7 @@ react:
 ## Overview
 Persistent state storage is particularly useful when running multiple instances of Handsontable on one page as it allows data separation per each instance.
 
-## Saving changes using a callback
+## Saving changes using a callbacks
```

Comment:
@maciejbo-handsontable sorry for setting you up, but in my 1f5c7151d1c1f24424c7e40b1f686573f039b2ff there was a typo. The word here should be `callbacks`

Reply:
Oops, I mean it should be `callback`. Sorry for confusion

Reply:
no problem - pushed